src/pages/DefaultOrgSelector/DefaultOrgSelector.jsx

```diff
@@ -88,25 +101,28 @@ function DefaultOrgSelector() {
     },
   })
 
-  const onSubmit = (data) => {
-    if (!data?.select) {
-      updateDefaultOrg({ username: currentUser?.user?.username })
-      return history.push(`/${provider}/${currentUser?.user?.username}`)
-    }
-
+  const onSubmit = () => {
     const segmentEvent = {
```

Comment:
Optional: we're getting rid of Segment, so it might be worth not adding this at all

Reply:
ohhh